Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/metempsychoticMachine.yml

```diff
@@ -0,0 +1,25 @@
+- type: entity
+  parent: CloningPod
+  id: MetempsychoticMachine
+  name: metempsychotic machine
+  description: Speeds along the transmigration of a soul to its next vessel.
+  components:
+    - type: MetempsychoticMachine
+    - type: CloningPod
+    - type: Machine
+      board: MetempsychoticMachineCircuitboard
+    - type: Sprite
+      sprite: Nyanotrasen/Structures/Machines/metempsychotic.rsi
+      snapCardinals: true
+      layers:
+        - state: pod_0
+    - type: Appearance
+    - type: GenericVisualizer
+      visuals:
+        enum.CloningPodVisuals.Status:
+          base:
+            Cloning: { state: pod_1 }
+            NoMind: { state: pod_1 }
+            Gore: { state: pod_1 }
+            Idle: { state: pod_0 }
+    - type: Psionic
```

Comment:
Everything that interacts directly with the Psionic system needs to have this component. In this context it's primarily for Metapsionic Pulse, and with the refactor that I'm actively coding also allows it to have unique flavor text when scanned with a focused Metapsionic pulse. To clarify, Psionic component does not imply that an entity is itself a psychic, merely stating that "This entity interacts with the Psionic systems in some way". With the exception of messing with Mantes, it is otherwise vestigial right now, I have left this component present because the original version of it from legacy code had the component.